### PR TITLE
SI: Add heat flux density (base unit watt per square meter, kg · s⁻³)

### PR DESCRIPTION
**Pull request by octronics**
Implemented from [Wikipedia][] data.

Wikipedia: https://en.wikipedia.org/wiki/SI_derived_unit#Examples_of_derived_quantities_and_units

### CODE REVIEW

**Comment by iliekturtles** [comment]
Thanks! I'm currently traveling so it may take me a bit to get through the review.

**Comment by octronics** [comment]
No problem.

File: src/si/heat_flux_density.rs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            test::<p::picowatt, d::picowatt_per_square_meter>();
+            test::<p::femtowatt, d::femtowatt_per_square_meter>();
+            test::<p::attowatt, d::attowatt_per_square_meter>();
+            test::<p::zeptowatt, d::zeptowatt_per_square_meter>();
+            test::<p::yoctowatt, d::yoctowatt_per_square_meter>();
+
+            fn test<P: p::Conversion<V>, D: d::Conversion<V>>() {
+                Test::assert_approx_eq(
+                    &HeatFluxDensity::new::<D>(V::one()),
+                    &(Power::new::<P>(V::one()) / (Length::new::<l::meter>(V::one()) * Length::new::<l::meter>(V::one()))));
```
Comment (iliekturtles) [inline src/si/heat_flux_density.rs]: Wrap and indent just before the `/` so the line doesn't go too long.

**Comment by octronics** [comment]
I don't understand your comment about renaming to `HeatFluxDensity`.

**Comment by iliekturtles** [comment]
Not sure what I was thinking. I was in the airport at the time and must have been pretty tired! PR merged. Thanks so much!

**Pull request merged**